package.json

```diff
@@ -24,6 +24,7 @@
     "@visx/pattern": "^1.7.0",
     "@visx/shape": "^1.7.0",
     "@xivanalysis/tooltips": "^0.1.3",
+    "caniuse-lite": "^1.0.30001285",
```

Comment:
I have reverted the commit since I don't see why this package was brought up
I re-run the PR in the test environment and no errors came up so this is good to remove